modules/apps/portal-settings/portal-settings-web/src/main/resources/META-INF/resources/default_portlet_decorator_configuration.jsp

```diff
@@ -0,0 +1,31 @@
+<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<aui:input name="<%= Constants.CMD %>" type="hidden" value="<%= Constants.UPDATE %>" />
+
+<div class="row">
+	<div class="col-md-12">
+		<br />
+
+		<aui:select label="default_portlet_decorator_configuration" name='<%= "settings--" + PropsKeys.DEFAULT_PORTLET_DECORATOR_ID + "--" %>' required="<%= true %>" type="text" value="<%= Validator.isNotNull((String)request.getAttribute(PropsKeys.DEFAULT_PORTLET_DECORATOR_ID)) ? (String)request.getAttribute(PropsKeys.DEFAULT_PORTLET_DECORATOR_ID) : PropsValues.DEFAULT_PORTLET_DECORATOR_ID %>">
```

Comment:
@torresvictor100, please remove the attributes `required` and `type`. In addition, please rename the `label` keeping only the name `default_portlet_decorator`.